<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgComponenteMaterial</class>
 <widget class="QDialog" name="dlgComponenteMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar componente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mksplanner.qrc">
    <normaloff>:/tableWindow/tools/material-3.png</normaloff>:/tableWindow/tools/material-3.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Material</string>
       </property>
       <property name="buddy">
        <cstring>cboMaterial</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cboMaterial">
       <property name="minimumContentsLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descripci√≥n</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblmaterialDecripcion">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cantidad necesaria</string>
       </property>
       <property name="buddy">
        <cstring>txtCantidad</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtCantidad"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Unidad de medida</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lblUnidadMedida">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mksplanner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgComponenteMaterial</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgComponenteMaterial</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
